src/containers/pages/FocusInvestigation/single/tests/index.test.tsx

```diff
@@ -101,43 +115,6 @@ describe('containers/pages/SingleFI', () => {
     wrapper.unmount();
   });
 
-  it('renders SingleFI correctly for null plan jurisdction id', () => {
```

Comment:
@p-netm was it necessary to completely remove this test?

Reply:
I believe so, it tested a case that we would not be supported any longer, i.e plans that had jurisdiction_ids with ' NULL' value.